eth/main.cpp

```diff
@@ -79,14 +79,9 @@ void help()
 	cout
 		<< "Usage eth [OPTIONS]" << endl
 		<< "Options:" << endl << endl
-		<< "Operating mode (default is non-interactive node):" << endl;
-#if !ETH_TRUE
-	cout
+		<< "Operating mode (default is non-interactive node):" << endl
 		<< "    import <file>  Import file as a concatenated series of blocks." << endl
-		<< "    export <file>  Export file as a concatenated series of blocks." << endl;
```

Comment:
They weren't commented out.   They were in an if !ETH_TRUE conditional.
Should they actually be deleted instead?

Yeah - I looked at all of these several times.    They do hurt your eyes, I know :-)